stairway/src/main/java/bio/terra/stairway/ProgressMeterData.java

```diff
@@ -0,0 +1,28 @@
+package bio.terra.stairway;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * POJO that represents the data for one project meter. A project meter is named and has two data
+ * values. The values are typically used to represent progress in the form of: N things done out of
+ * M total things.
+ */
+public class ProgressMeterData {
```

Comment:
What's the distinction between `ProgressMeter` and `ProgressMeterData`? If there's none, it would be less wordy and just as clear to call this `ProgressMeter`.

Reply:
Good point. I had split things differently and then redid them. I renamed it to ProgressMeter.